js_modules/dagit/packages/core/src/asset-graph/useLiveDataForAssetKeys.tsx

```diff
@@ -15,7 +20,8 @@ import {AssetGraphLiveQuery, AssetGraphLiveQueryVariables} from './types/AssetGr
  */
 export function useLiveDataForAssetKeys(
   pipelineSelector: PipelineSelector | null | undefined,
-  graphData: GraphData | null,
+  assets: AssetDefinitionsForLiveData | undefined,
+  graphForUpstreamChanged: GraphData | null,
```

Comment:
Kind of an odd assortment of `null`, and `undefined` possibilities.

Reply:
Ahh good call, will fix